client/src/components/AdminCard/index.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Button, Pagination } from 'antd';
+import './style.css';
+
+const AdminCard = ({
+  imgUrl,
+  name,
+  description,
+  githbUrl,
+  student,
+  websiteLink,
+  studentNames,
+  ComProject,
+  CliProject,
+  editCard,
+  deleteCard,
+}) => {
+  return (
+    <>
+      <section className="card-container">
+        <ul>
+          <li>
+            <div className="name-img-card">
+              <div className="img-card">
+                <img src={imgUrl} alt="card" />
+              </div>
+              <div>
+                <h3>Name</h3>
+                <p>{name}</p>
+              </div>
+            </div>
+            {description && (
+              <div>
+                <h3>Description</h3>
+                <p>{description}</p>
+              </div>
+            )}
+            <div>
+              <h3>Github link</h3>
+              <NavLink exact to={githbUrl}>
+                Click
+              </NavLink>
+            </div>
+            {student && (
+              <div>
+                <h3>Student</h3>
+                <NavLink exact to="/admin/students">
```

Comment:
as I mentioned above between the differences between ``` <Nav Link> ``` and ``` <Link> ``` I think you don't need use ``` <NavLink> ``` here and you can use ``` <Link> ``` like this :
```js
<Link to="/admin/students">
```
don't forget to remove ``` exact ``` beacuse it does't exist in ``` <Link> ```
[For more details](https://reacttraining.com/react-router/web/api/Link)

Reply:
👍